lib/supportFeature.js

```diff
@@ -1,4 +1,11 @@
 const features = require('./features')
 
-module.exports = (feature, version) =>
-  features.some(({ name, versions }) => name === feature && versions.includes(version))
+module.exports = (feature, minecraftVersion) => {
+  const f = features.find(f => f.name === feature)
+
+  const currentVer = require('minecraft-data')(minecraftVersion).version.version
+  const minVer = require('minecraft-data')(f.versions[0]).version.version
```

Comment:
Isn't this version the protocol version ? It would be better not to use that and instead use the position of the version in the array. It's better to avoid relying on this protocol version as mojang rules for updating it are not consistent.